hr_employee_calendar_planning/models/hr_employee_calendar.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
```

Comment:
Why splitting this in a new file? It was put on purpose in the previous one. Check contributing guidelines where this is specified that a submodel should go in the same file of the main model:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming

Reply:
Okay! Fixed.